corehq/apps/users/tasks.py

```diff
@@ -58,7 +58,7 @@ def tag_forms_as_deleted_rebuild_associated_cases(user_id, domain, form_id_list,
     cases_to_rebuild = set()
 
     for form in FormAccessors(domain).iter_forms(form_id_list):
-        if form.domain != domain:
+        if form.domain != domain or form.is_error:
```

Comment:
Yeah. I had thought about that, but I wasn't sure if anything not normal could change the case. seemed safer to just remove errored forms from this